src/serveur/services/i18n.service.ts

```diff
@@ -55,8 +54,10 @@ class I18nService {
 					content = content.replaceAll(match[0], substitutions[index - 1]);
 				}
 			}
+			return content
+		} else {
+			console.warn(`A character string is incorrect for this message : ${message}.`);
```

Comment:
Je serai plus explicite sur la message d’erreur, pour signifier qu’une partie du message contient `undefined`. Genre `Something in this message is undefined: ${message}`. Et peut-être même boucler sur les substitutions s’il y en a, pour les mentionner dans le message ?

Qu’en dis-tu ?